bazel/ray_deps_setup.bzl

```diff
@@ -1,131 +1,194 @@
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository", "new_git_repository")
 load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 
-def ray_deps_setup():
-    RULES_JVM_EXTERNAL_TAG = "2.10"
-    RULES_JVM_EXTERNAL_SHA = "1bbf2e48d07686707dd85357e9a94da775e1dbd7c464272b3664283c9c716d26"
+def github_repository(*, name=None, remote=None, commit=None, tag=None,
+                      branch=None, build_file=None, build_file_content=None,
+                      sha256=None, shallow_since=None, strip_prefix=True,
+                      url=None, path=None, **kwargs):
+    """
+    Conveniently chooses between archive, git, etc. GitHub repositories.
+    Prefer archives, as they're smaller and faster due to the lack of history.
+
+    One of {commit, tag, branch} must also be provided (as usual).
+
+    sha256 should be omitted (or None) when the archive hash is unknown, then
+    updated ASAP to allow caching & avoid repeated downloads on every build.
+
+    NOTE: False and None mean different things.
```

Comment:
don't need this, it's a python convention :)

Reply:
Alrighty :P